projects/Mallard/src/screens/settings/manage-editions-screen.tsx

```diff
@@ -1,38 +1,156 @@
+import { useApolloClient, useQuery } from '@apollo/react-hooks'
+import gql from 'graphql-tag'
 import React from 'react'
-import { Alert } from 'react-native'
+import { Alert, Switch, TouchableOpacity, View, StyleSheet } from 'react-native'
 import { List } from 'src/components/lists/list'
+import { UiBodyCopy } from 'src/components/styled-text'
 import { deleteIssueFiles } from 'src/helpers/files'
+import {
+    setMaxAvailableEditions,
+    setWifiOnlyDownloads,
+} from 'src/helpers/settings/setters'
 import { WithAppAppearance } from 'src/theme/appearance'
 
+const buttonStyles = StyleSheet.create({
+    background: {
+        borderWidth: 1,
+        borderRadius: 3,
+        flex: 1,
+        marginHorizontal: 5,
+        padding: 10,
+        alignItems: 'center',
+    },
+})
+
+const MultiButton = ({
```

Comment:
I'm, happy with this being in here. It's an implementation detail of this component that is not really useful anywhere else as it stands. If it ever gets a refactor to be reusable then it can come out then.